manifests/a/AVerMedia/RECentral/4.7.44.1/AVerMedia.RECentral.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AVerMedia.RECentral
+PackageVersion: 4.7.44.1
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft

```

Comment:
Installer does not install silently in a sandbox environment; Silent switches must be provided for the package